active_model_otp.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rotp", "~> 5.0.0"
 
   spec.add_development_dependency "activerecord"
+  spec.add_development_dependency "activemodel-serializers-xml"
```

Comment:
The motivation behind this change is the error I've been facing during the development process.
```
.../active_model_otp/test/models/user.rb:4:in `<class:User>': uninitialized constant ActiveModel::Serializers::Xml (NameError)
```